tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -149,16 +149,14 @@ bool IsXlaGlobalJitOn(
 
 // A helper function to decide whether to enable the memory optimizer.
 bool MemoryOptimizerEnabled(
-    RewriterConfig::MemOptType mem_opt_type,
-    OptimizerOptions::GlobalJitLevel jit_level_in_session_opts) {
+    RewriterConfig::MemOptType mem_opt_type, bool xla_on) {
```

Comment:
Boolean parameters are generally discouraged, can we instead check whether we are inside XLA context? cf. https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/python/ops/control_flow_ops.py;l=1754?q=xlacontext%20file:tensorflow

Reply:
George, NVIDIA folks are probably trying to get this to work with auto-clustering, not `jit_compile`.